src/dotnet/APIView/APIViewWeb/Repositories/CommentsManager.cs

```diff
@@ -47,12 +88,22 @@ public async Task AddCommentAsync(ClaimsPrincipal user, CommentModel comment)
             }
         }
 
-        public async Task<CommentModel> UpdateCommentAsync(ClaimsPrincipal user, string reviewId, string commentId, string commentText)
+        public async Task<CommentModel> UpdateCommentAsync(ClaimsPrincipal user, string reviewId, string commentId, string commentText, string[] taggedUsers)
         {
             var comment = await _commentsRepository.GetCommentAsync(reviewId, commentId);
             await AssertOwnerAsync(user, comment);
             comment.EditedTimeStamp = DateTime.Now;
             comment.Comment = commentText;
+
+            HashSet<string> newTaggedUsers = new HashSet<string>();
+            foreach(string taggedUser in taggedUsers)
+            {
+                if(!comment.TaggedUsers.Contains(taggedUser))
+                {
+                    // TODO: notify users they have been tagged
+                }
+                newTaggedUsers.Add(taggedUser);
+            }
```

Comment:
My guess is comment.taggedUsers should be updated with additional new users.

Reply:
Yes - I planned to finish this when handling the user notification portion, but as that was moved to #4164 I forgot - fixed now!